ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -366,4 +384,8 @@ export class WorkflowDetails extends React.Component<RouteComponentProps<any>, {
     private get appContext(): AppContext {
         return this.context as AppContext;
     }
+
+    private openLink(link: Link) {
```

Comment:
This is great, just a minor suggestion, I think a new window/tab would be better.

Reply:
good idea, I prefer ctrl+click to open a new window - I'll check that works

Reply:
ok, we can't do ctrl+click :(